app/helpers/react_components/student/request_mentoring_button.rb

```diff
@@ -0,0 +1,30 @@
+module ReactComponents
+  module Student
+    class RequestMentoringButton < ReactComponent
+      def to_s
+        super(
+          "student-request-mentoring-button",
+          {
+            request: request,
+            links: {
+              mentor_request: Exercism::Routes.new_track_exercise_mentor_request_path("$TRACK_SLUG", "$EXERCISE_SLUG")
+            }
+          }
+        )
+      end
+
+      private
+      def request
+        {
+          endpoint: Exercism::Routes.api_solutions_url,
+          query: {
+            # TODO: Update API to accept arrays
```

Comment:
@iHiD The API currently doesn't accept arrays for `status` and `mentoring_status`.